admin/provisioner/clickhousestatic/provisioner.go

```diff
@@ -22,15 +22,11 @@ func init() {
 }
 
 type Spec struct {
-	// DSN with admin permissions for a Clickhouse service.
-	// This will be used to create a new (virtual) database and access-restricted user for each provisioned resource.
-	DSN string `json:"dsn"`
-	// Path to a file that we should load the DSN from.
-	// This is an alternative to specifying the DSN directly, which can be useful for secrets management.
-	DSNPath string `json:"dsn_path"`
```

Comment:
I think only `dsn_path` can be removed, not `dsn`. The `dsn` setting is used in the local dev environments:
```
"clickhouse": {"type": "clickhouse-static", "spec": {"dsn": "clickhouse://default:default@localhost:9000"}}
```